### PR TITLE
Fixes bug 1195310 - Showed startup crash icon in new topcrashers.

### PR DESCRIPTION
**Pull request by AdrianGaudebert**
@peterbe r?

### CODE REVIEW

File: webapp-django/crashstats/topcrashers/views.py [hunk 1]
```diff
@@ -136,6 +138,16 @@ def get_topcrashers_results(**kwargs):
                 if row['term'] in (1, -1):
                     hit['hang_count'] += row['count']
 
+            # Number of startup crashes.
+            hit['startup_percent'] = 0
+
+            sig_startup = hit['facets']['histogram_uptime']
+            for row in sig_startup:
+                if row['term'] == 0:
+                    hit['startup_percent'] = (
```
Comment (peterbe) [inline webapp-django/crashstats/topcrashers/views.py]: Is there any other purpose of having this variable? 
In the django template, you could ideally move some of the business logic in here in the view code. E.g.
```python
startup_ratio = 1.0 * row['count'] / hit['count']
# here goes some comment that explains why 0.5 (aka 50%) is used and hardcoded in the code. 
hit['startup_crash'] = startup_ratio > 0.5
```

File: webapp-django/crashstats/topcrashers/views.py [hunk 2]
```diff
@@ -136,6 +138,16 @@ def get_topcrashers_results(**kwargs):
...
+                        100.0 * row['count'] / hit['count']
```
Comment (peterbe) [inline webapp-django/crashstats/topcrashers/views.py]: Why don't you have to worry about `hit['count'] != 0`?

**Comment by peterbe** [comment]
Seems to work. I get a bunch of rockets using stage data. 

Some stylistic things to consider about the 50% thing being put into the template code. 

r+

File: webapp-django/crashstats/topcrashers/views.py (hunk 1 above)
Comment (AdrianGaudebert) [inline webapp-django/crashstats/topcrashers/views.py]: Yep, that's right. I just copied the way it was done in the old TCBS, but yours is better.

File: webapp-django/crashstats/topcrashers/views.py (hunk 2 above)
Comment (AdrianGaudebert) [inline webapp-django/crashstats/topcrashers/views.py]: Super Search won't return any result where ``count`` is zero.

**Pull request merged**